general/g.access/exp_perms.c

```diff
@@ -4,31 +4,31 @@
 char *explain_perms(int group, int other, int will)
 {
     static char buf[256];
-    char *who;
-    char *verb;
-    char *read;
+    char *msg = "Cannot explain access permissions for mapset %s";
+
+    msg = _("Cannot explain access permissions for mapset %s");
```

Comment:
This can be simplified with:
```suggestion
    char *msg = _("Cannot explain access permissions for mapset <%s>");
```

Note: it is custom practice to put the name of mapset in angular brackets. Please update below.

Reply:
> This can be simplified with:
> 
> ```suggestion
> 
>     char *msg = _("Cannot explain access permissions for mapset <%s>");
> 
> ```
> 
> 
> 
> Note: it is custom practice to put the name of mapset in angular brackets. Please update below.

Done. If I ever refind the strings that made me start to compare raster and raster3d implementations, I'll ask you again what is the best practice. There was one string with `<%s@%s>`, and the other one was more as text, and both versions were about 15 years old, so I couldn't determine which one was better than the other.